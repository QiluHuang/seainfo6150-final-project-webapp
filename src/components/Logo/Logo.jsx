import React from 'react';
// import UncleWongLogo from "./images/uncle-wong-logo.gif";
import UncleWongLogo from "./uncle-wong-logo.gif";
import styles from './Logo.module.css';

const Logo = () => {
    return (
        <img className={styles.logo} src={UncleWongLogo} alt="Uncle Wong Logo" />
    );
}

export default Logo;