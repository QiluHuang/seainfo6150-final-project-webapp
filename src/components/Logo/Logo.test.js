import React from "react";
import { render } from "@testing-library/react";
import Logo from "./Logo.jsx";

describe("Logo tests", () => {
    it("renders the logo component correctly", () => {
    const { container } = render(<Logo />);
    expect(container).toMatchSnapshot();
    });
});