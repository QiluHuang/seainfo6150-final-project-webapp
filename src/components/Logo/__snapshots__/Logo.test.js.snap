// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo tests renders the logo component correctly 1`] = `
<div>
  <img
    alt="Uncle Wong Logo"
    class="logo"
    src="uncle-wong-logo.gif"
  />
</div>
`;
